pkg/cloud/cloud.go

```diff
@@ -70,6 +101,8 @@ func getAssetsConstructor(platformStatus *configv1.PlatformStatus) (assetsConstr
 		return gcp.NewProviderAssets, nil
 	case configv1.IBMCloudPlatformType:
 		return ibm.NewProviderAssets, nil
+	case configv1.OpenStackPlatformType:
```

Comment:
Is this to make it alphabetical? If so, 👍

Reply:
Yup. Some non-foolish consistency, hopefully :wink: